docs/intro/tutorial.rst

```diff
@@ -323,34 +323,69 @@ use Firefox Developer Tools or some Firefox extensions like Firebug. For more
 information see :ref:`topics-firebug` and :ref:`topics-firefox`.
 
 After inspecting the page source, you'll find that the web site's information
-is inside a ``<ul>`` element, in fact the *second* ``<ul>`` element.
-
-So we can select each ``<li>`` element belonging to the site's list with this
+is inside ``<div>`` elements. In fact, the title and description of the site's list
+are included in ``<div>`` elements that contain the ``class`` attribute
+``"title-and-desc"``.::
+
+    <div id="sites-section" class="section-wrapper">
+        <div id="site-list-content" class="results browse-content">
+        
+            <div class="site-item ">
+                <div class="site-icon "><i class="siteIcon fa fa-file-text-o"></i></div>
+                <div class="site-flag" style="display: none;">
+                   <a href="/public/flag?t=computers&amp;cat=Computers%2FProgramming%2FLanguages%2FPython%2FBooks&amp;url=http%3A%2F%2Fwww.pearsonhighered.com%2Feducator%2Facademic%2Fproduct%2F0%2C%2C0130260363%2C00%252Ben-USS_01DBC.html" title="Report an issue with this listing"><i class="fa fa-flag"></i></a>
+                </div>
+    
+                <div class="title-and-desc">
+                   <a target="_blank" href="http://www.pearsonhighered.com/educator/academic/product/0,,0130260363,00%2Ben-USS_01DBC.html">
+                     <div class="site-title" style="text-decoration: none;">Core Python Programming </div>
+                   </a>
+                   <div class="site-descr ">
+			    
+                        By Wesley J. Chun; Prentice Hall PTR, 2001, ISBN 0130260363. For experienced developers to improve extant skills; professional level examples. Starts by introducing syntax, objects, error handling, functions, classes, built-ins. [Prentice Hall]
+                        <span class="content-type media-date"></span>
+                   </div>
+                </div>
+            </div>
+
+So we can select each ``<div>`` element belonging to the site's list with this
 code::
 
-    response.xpath('//ul/li')
+	response.xpath('//div[@id="site-list-content"]/div[@class="site-item "]')
+
+You can also use CSS selectors as the expression is much simpler::
+
+	response.css('#site-list-content > div.site-item')
 
 And from them, the site's descriptions::
 
-    response.xpath('//ul/li/text()').extract()
+	response.xpath('//div[@id="site-list-content"]/div[@class="site-item "]/div[@class="title-and-desc"]/div[@class="site-descr "]/text()').extract()
```

Comment:
Here is a good case for introducing whitespaced XPaths, to bring a bit of air.
I tend to like triple-quoted strings, i.e:

```
response.xpath('''//div[@id="site-list-content"]
                        /div[@class="site-item "]
                            /div[@class="title-and-desc"]
                                /div[@class="site-descr "]
                                    /text()''').extract()
```